tmt/base.py

```diff
@@ -548,14 +548,16 @@ def create(name, template, tree, force=False):
             path=plan_path, content=content,
             name='plan', dry=dry, force=force)
 
-    def steps(self, enabled=True, disabled=False, names=False, skip=[]):
+    def steps(self, enabled=True, disabled=False, names=False, skip=None):
         """
         Iterate over enabled / all steps
 
         Yields instances of all enabled steps by default. Use 'names' to
         yield step names only and 'disabled=True' to iterate over all.
         Use 'skip' to pass the list of steps to be skipped.
         """
+        if skip is None:
```

Comment:
`skip = skip or None`

is shorter and nice :)

Reply:
Yeah, I just remember that it can be written as `skip = skip or []`... Will fix